app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+  def new
+    @user = User.new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      flash[:success] = t(".welcome")
+      redirect_to @user
+    else
+      render "new"
```

Comment:
```suggestion
      render :new 
```
e thêm 1 flash.now[:danger] báo lỗi nhé